etc/ciao/ciao.json

```diff
@@ -0,0 +1,9 @@
+{
+    "server": "localhost3",
+    "cacert": "/etc/pki/ciao/CAcert-server-localhost.pem",
+    "cert": "/etc/pki/ciao/cert-client-localhost.pem",
+    "compute-net":  "",
+    "mgmt-net":  "",
+    "disk-limit": true,
+    "mem-limit": true
+}
```

Comment:
How do you plan to support controller, launcher (NN and CN) and scheduler from the same config file ? Each block should be namespaced.